docs/transcription.md

```diff
@@ -14,43 +14,106 @@ Depending on the needs of customers, you can define individual prices. You can a
 
 <img src="/transcription/img/transcription-enable-transcriptions.png" enable transcription="package" width="100"/>
 
-### Create a Transcription Package
+!!! Note
+    The Transcription service fee is **$0.0003** USD per minute.
 
-To create a package to resell to your customers, go to Config :material-menu-right: [Packages](https://docs.connexcs.com/customer/package/). Click on <img src="/transcription/img/transcription-add.png" alt="add" width="50"/> and choose your Transcription Package from **ConnexCS Package**.
+!!! Warning
+    Please contact us for assistance and enable the transcription service if you are unable to find the **Enable Transcription** option.
 
-Select your **Retail Cost** and click on <img src="/transcription/img/transcriptions-save.png" alt="save" width="120"/> and choose your Transcription Package from **ConnexCS Package**.
+!!! Info
+    Please take note that there is **no fee** for **silence**.
 
-<img src="/transcription/img/transcription-package.png" alt="package" width="500"/>
+    It implies that if a call lasts for 50 seconds and no audio is exchanged for 20 seconds, we will make every effort to cut out the silence from the audio and bill you for 30 seconds of transcribing service.
 
-### Automated Transcriptions
+### Add Transcription Service for your Customer
 
-Add Package to Customers Account, assign the **Quantity**, and also assign a **Start Date** and **End date** to the package. At the End Date, the package will automatically stop working.
+Once this service has been enabled on your account, you can follow the steps below to enable this for your customers:
 
-<img src="/transcription/img/transcriptions-automated-transcriptions.png" alt="automated transcript" width="400"/>
+1. Navigate to **Management :material-menu-right: Customer :material-menu-right: [Customer Name] :material-menu-right: Routing :material-menu-right: Ingress Routing  :material-menu-right: Media :material-menu-right: Transcribe**.
+2. From the drop-down you can select from multiple options like **Disabled**, **1% Sampling**, **5% Sampling**, **25% Sampling**, **50% Sampling** and **Enabled (Always On)**.
 
-Click on <img src="/transcription/img/transcriptions-save.png" alt="save" width="120"/>, and the package will be available with the customer.
+!!! Info
+    Out of all the calls that are accessible, **% Sampling** indicates the proportion of calls that will be transcribed.
 
-In Routing :material-menu-right: Media, you can enable or disable this service.
+<img src="/transcription/img/trans1.png" alt="enable" width="900"/>
 
-<img src="/transcription/img/transcription-enable-disable.png" alt="enable" width="900"/>
+3.**Transcription Duration**: You can set a maximum transcription time (in seconds) for your transcription service per call. Please note, the transcription will stop at the specified time, even if mid-sentence.
```

Comment:
will stop at the specified time, even if it is mid-sentence.

Reply:
done